### ISSUE TITLE
Configuration option for different api routes not using /api prefix

### ISSUE THREAD

**Issue by heisig** [harlan-zw/nuxt-simple-sitemap#42]
### Clear and concise description of the problem

It would be great to use other routes than `/api/_sitemap-urls` for handling dynamic URLs.
In our project, the routes with prefix `/api` are used differently and collide with the hardcoded `server/api/_sitemap-urls.ts`.

### Suggested solution

If possible, a configuration could be implemented to choose the prefered route.
[Here](https://nuxt.com/docs/guide/directory-structure/server#server-routes) the nuxt documentation describes how to use server routes without the `/api` prefix

Maybe this route has to be configurable too: [`/api/__sitemap__/document-driven-urls`](https://github.com/harlan-zw/nuxt-simple-sitemap/blob/b58847f47e93c90b3c4d787cea670b52ae61eb58/src/module.ts#L257)

I would love to hear your thoughts on this topic.

**Comment by harlan-zw** [comment]
Hey @heisig, thanks for the suggestion!

Would you mind reviewing the PR to support it: https://github.com/harlan-zw/nuxt-simple-sitemap/pull/44

I'm hesitant to modify the `/api/__sitemap__/document-driven-urls` endpoint as it's one registered by the module. Is your site actualy using it? If so, can you elaborate on the issues you have with this? Is it because you're applying some middleware to all api routes?

**Comment by harlan-zw** [comment]
Thanks for the details :) 

The PR is merged and available in 2.5.0. Give it a try and if you're still having issues happy to investigate further configuration requirements

```ts
export default defineNuxtConfig({
  sitemap: {
    dynamicUrlsApiEndpoint: '/__sitemap',
  },
})
```
